experimental/python/databricks/bundles/jobs/__init__.py

```diff
@@ -96,6 +105,11 @@
     "JobParameterDefinition",
     "JobParameterDefinitionDict",
     "JobParameterDefinitionParam",
+    "JobPermission",
```

Comment:
From your summary:
> Rename jobs.Permission into jobs.JobPermission to avoid confusion, following the spec.

I don't fully understand the mechanics here yet, how is this generated? And is there a `README` covering that for maintainers somewhere?

Reply:
`JobPermission `is fixed in in https://github.com/databricks/cli/pull/2642, I've just generated the code using effective jsonschema